ReleaseNotes.md

```diff
@@ -1,92 +1,77 @@
-* [FLUID-5582: Improve flexibility of `fluid.fail` handling further](https://issues.fluidproject.org/browse/FLUID-5582)
-* [FLUID-5667: Circularity in material sent for options expansion is not detected](https://issues.fluidproject.org/browse/FLUID-5667)
-* [FLUID-5711: The Infusion core framework doesn't work when run in a Web Worker](https://issues.fluidproject.org/browse/FLUID-5711)
-* [FLUID-5509: Storing NaN via a "new-style applier" will cause infinite recursion in DataBinding](https://issues.fluidproject.org/browse/FLUID-5509)
-* [FLUID-5675: Text To Speech enactor should be supplied with a mock implementation to facilitate integration testing](https://issues.fluidproject.org/browse/FLUID-5675)
+* Constraint-based priorities, supported by `listeners`, `modelListeners`, `modelRelay`, `distributeOptions`, `contextAwareness`, and `components`
+* Context Awareness - and things it relies on:
+  * Global Instantiator
+    * Every Infusion component, regardless of how it is instantiated, ends up in a single-rooted tree of components
+    * This enables use of modern IoC features such as model relay and declarative event binding
+    * Enables use of the root distributeOptions context "/"
+    * Enables the removal of "demands blocks"
+    * Useful debugging tip: Watch `fluid.globalInstantiator` in your JS debugging tools to see the structure of your application and its tree.
+* `fluid.notImplemented` function for implementing abstract grades
+* [Lazy loading UI Options](http://docs.fluidproject.org/infusion/development/UserInterfaceOptionsAPI.html#lazyload) and instructions for how to use the Prefernces Framework with a [minimal footprint](http://docs.fluidproject.org/infusion/development/tutorial-prefsFrameworkMinimalFootprint/MinimalFootprint.html).
+  * This should assist in improving performance when using the Preferences Framework, particularly for resource intensive sites and applications
+* Much faster invokers and boiled listeners (c. 60x faster)
+* NPM - "dists" directory containing pre-built versions of Infusion
+* Source Maps are generated for the concatenated JavaScript files
+* View oriented IoC debugging tools
+  * FluidViewDebugging.js - include this at the head of your HTML file to get access to the "IoC inspector"
 
 ### Removals
 
-* [FLUID-5616: "init functions" removed from the framework](https://issues.fluidproject.org/browse/FLUID-5616)
-* Also removed: the static and dynamic environments, dynamic invokers, old ChangeApplier, autoInit, etc.
+* Manual lifecycle points finalInit, postInit, etc.
+* Obsolete syntax for arguments, options, etc.
+* `"autoInit"` grade
+* Static and dynamic environments, replaced by Global Instantiator
+* The old model component hierarchy and "old ChangeApplier" implementation
+* `fluid.demands`
+* No more distinction between fast and dynamic invokers
+* Model Relay specific component grades have been removed, model relay now works with any model grade.
+
 
 ## What's New in 1.9 ##
```

Comment:
Typically we wouldn't want a section for a prior release in the release notes. However, we didn't actually release 1.9 so this includes those changes. We could move this info up to the What's new in 2.0 section though. What do you think?